Basic/ClientDriven/Assets/Scripts/NetworkObjectSpawner.cs

```diff
@@ -0,0 +1,30 @@
+using System;
+using Unity.Netcode;
+using UnityEngine;
+using UnityEngine.Networking;
+using Random = System.Random;
+
+public class NetworkObjectSpawner : NetworkBehaviour
+{
+    public NetworkObject prefabReference;
+    Random m_RandomGenerator = new Random();
+
+    public override void OnNetworkSpawn()
+    {
+        base.OnNetworkSpawn();
+        if (!IsServer)
+        {
+            enabled = false;
+            return;
+        }
```

Comment:
1. This code is unneeded if you simply put the `enabled = false` after (and outside) the entire `if(isServer)` that comes next